dvc/utils/pager.py

```diff
@@ -14,9 +14,33 @@
 
 
 DEFAULT_PAGER = "less"
-DEFAULT_PAGER_FORMATTED = (
-    f"{DEFAULT_PAGER} --chop-long-lines --clear-screen --RAW-CONTROL-CHARS"
-)
+LESS = "LESS"
+PAGER_ENV = "PAGER"
+
+
+def prepare_default_pager(
+    clear_screen: bool = False,
+    quit_if_one_screen: bool = True,
+    ansi_escapes: bool = True,
+    chop_long_lines: bool = True,
+    no_init: bool = True,
+    no_tilde: bool = False,
+) -> str:
+    args = [DEFAULT_PAGER]
+    if clear_screen:
```

Comment:
the only question- do all versions of less have these arguments? (e.g. GNU vs BSD)?

Reply:
@shcheklein. the new flags that we have started to use with this PR is `--quit-if-one-screen/-F` and `--no-init/-X` (introduced ~2000 as per the [changelog](https://github.com/gwsw/less/blob/master/NEWS.VER)). And, git already defaults to `-FRX` so that's another point for the confidence. `-S` seems to be there forever, and the latest change was on `-c` which had a behavior change in 2005 (which we are not using now).

Reply:
yep, sorry my question was not exactly clear. It was not about versions in terms of older versions vs newer, it's about different implementations (like `grep` for example has a bit different arguments on Mac and Linux). Anyways, your point about `And, git already defaults to -FRX so that's another point for the confidence` makes it look safe.

Reply:
@shcheklein, I don't think there are variants of `less` at all, the versions might be different though. I tested the one that comes in macOS (less 487 2016 version) and the latest one (less 581 2021 version).